executor/inspection_result.go

```diff
@@ -316,101 +316,114 @@ func (versionInspection) inspect(_ context.Context, sctx sessionctx.Context, fil
 	return results
 }
 
-func (c currentLoadInspection) inspect(_ context.Context, sctx sessionctx.Context, filter inspectionFilter) []inspectionResult {
-	var commonResult = func(item, expected string, row chunk.Row) inspectionResult {
-		return inspectionResult{
-			tp:       row.GetString(0),
-			instance: row.GetString(1),
-			item:     item,
-			actual:   row.GetString(2),
-			expected: expected,
-			severity: "warning",
-		}
+func (c nodeLoadInspection) inspect(ctx context.Context, sctx sessionctx.Context, filter inspectionFilter) []inspectionResult {
+	var rules = []ruleChecker{
+		inspectCPULoad{item: "load1", tbl: "node_load1"},
+		inspectCPULoad{item: "load5", tbl: "node_load5"},
+		inspectCPULoad{item: "load15", tbl: "node_load15"},
+		inspectVirtualMemUsage{},
+		inspectSwapMemoryUsed{},
+		inspectDiskUsage{},
 	}
-	var diskResult = func(item, expected string, row chunk.Row) inspectionResult {
-		return inspectionResult{
-			tp:       row.GetString(0),
-			instance: row.GetString(1),
-			item:     item,
-			actual:   row.GetString(3),
-			expected: expected,
-			severity: "warning",
-			detail: fmt.Sprintf("current disk-usage is too high, execute the sql to see more detail: select * from information_schema.cluster_hardware where type='%s' and instance='%s' and device_type='disk' and device_name='%s'",
-				row.GetString(0), row.GetString(1), row.GetString(2)),
-		}
+	return checkRules(ctx, sctx, filter, rules)
+}
+
+type inspectVirtualMemUsage struct{}
+
+func (inspectVirtualMemUsage) genSQL(timeRange plannercore.QueryTimeRange) string {
+	sql := fmt.Sprintf("select instance, max(value) as max_usage from metrics_schema.node_memory_usage %s group by instance having max_usage >= 70", timeRange.Condition())
+	return sql
+}
+
+func (i inspectVirtualMemUsage) genResult(sql string, row chunk.Row) inspectionResult {
+	return inspectionResult{
+		tp:       "node",
+		instance: row.GetString(0),
+		item:     i.getItem(),
+		actual:   fmt.Sprintf("%.1f%%", row.GetFloat64(1)),
+		expected: "< 70%",
```

Comment:
Should we change it to `threshold ：“70%“,`

Reply:
No, some rule the check is ` < threshold value`, but some rule check is `> threshold value`.